### PR TITLE
Updated get started and generic procedures for types and streams

### PR DESCRIPTION
**Pull request by lasato-OSI**
Hi Dave,

This PR includes the generic procedures for types and streams as well as the get started topics. You're welcome to review all topics or you can ignore the get started topics.

Thanks,
Laureen

### CODE REVIEW

**Review by dwicinas-osisoft — APPROVED** [review]
I did not review the getting started topics. The other topics looked good!

**Review by joyceseng — APPROVED** [review]
Not sure I did this correctly! Buyt since all my edits to types-procedure were style (not technical content) i made changes directly in the file, rather than leaving comments. Hope that is ok!

**Review by VictoriaTouati-AVEVA — COMMENTED** [review]
Comments for gs-streams.md:

Step 1: Menu icon graphic did not display when I viewed it. Check link.
Step 2: Check if 2nd graphic renders correctly on local build. 3rd P, change "steams" to "streams".
Step 4: 2nd P, change "This opens..." to "Opens...". 4th P, Change "the permissions" to "permissions".
Step 7: Put "(Optional)" in front of step after number.
Step 12: 2nd P, Change "This searches for streams..." to "This action searches for streams...".

File: content/overview/get-started/gs-streams.md [hunk 1]
```diff
@@ -14,9 +14,13 @@ After creating your types, you can add streams and associate a type with the str
 
 1. Click the ![Menu icon](images\menu-icon.png) and click **Sequential Data Store** (under Data Management).
```
Comment (lasato-OSI) [inline content/overview/get-started/gs-streams.md:15]:
```suggestion
1. Click the ![Menu icon](images/menu-icon.png) and click **Sequential Data Store** (under Data Management).
```

File: content/overview/get-started/gs-streams.md [hunk 2]
```diff
@@ -14,9 +14,13 @@ After creating your types, you can add streams and associate a type with the str
...
+    ![Namespace list](images/sds-gs-namespace.png)
 
-1. Click the **Manage Default Stream Permissions** icon above the tree on the left.  
+1. From the **Streams** drop-down list, select **Streams** if it is not already selected. 
+    ![Streams list](images/streams-list.png)
+
+1. Click the ![More options](images/more-options.png) icon on the menu bar and click **Manage Default Permissions**.  
     This opens the `Manage Default Permissions for New Streams` window where default permissions for streams created in the selected namespace are configured. 
    
     **Note:** By default, you must be assigned the Tenant Administrator role to change the default permissions. Any changes only affect new streams. It does not change the permissions on already existing steams.
```
Comment (lasato-OSI) [inline content/overview/get-started/gs-streams.md:26]:
```suggestion
    **Note:** By default, you must be assigned the Tenant Administrator role to change the default permissions. Any changes only affect new streams. It does not change the permissions on already existing streams. 
```

File: content/overview/get-started/gs-streams.md [hunk 3]
```diff
@@ -14,9 +14,13 @@ After creating your types, you can add streams and associate a type with the str
...
-1. Click **Streams**. From the **Namespace** drop-down list, select **MyOCS**.
+1. Click the **Namespace** selection button to open the `Select a Namespace` window. Select the namespace in which you want to create the stream. 
+    ![Namespace list](images/sds-gs-namespace.png)
 
-1. Click the **Manage Default Stream Permissions** icon above the tree on the left.  
+1. From the **Streams** drop-down list, select **Streams** if it is not already selected. 
+    ![Streams list](images/streams-list.png)
+
+1. Click the ![More options](images/more-options.png) icon on the menu bar and click **Manage Default Permissions**.  
     This opens the `Manage Default Permissions for New Streams` window where default permissions for streams created in the selected namespace are configured.
```
Comment (lasato-OSI) [inline content/overview/get-started/gs-streams.md:24]:
```suggestion
    The `Manage Default Permissions for New Streams` window opens. The default permissions for streams created in the selected namespace are configured here.
```

File: content/overview/get-started/gs-streams.md [hunk 4]
```diff
@@ -27,37 +31,34 @@ After creating your types, you can add streams and associate a type with the str
...
 
-1. In `Enter Metadata key`, enter **Site**, and in `Enter Metadata value`, enter **SLTC**. 
-
-    **Note:** You can click either of these fields to edit them later, click the **X** to delete it from the stream, or click the ![Info icon](images/Info.png)to see who last changed this metadata key.
+1. Click **Save**.
 
-1. In the `Search` field, enter **Site:SLTC**. 
+1. In the `Search` field, enter **Site:SLTC**
 
     This searches for streams that have the metadata key *Site* and the value *SLTC*. This search query returns the MyOCS stream.
```
Comment (lasato-OSI) [inline content/overview/get-started/gs-streams.md:56]:
```suggestion
    OCS searches for streams that have the metadata key *Site* and the value *SLTC*. This search query returns the MyOCS stream. 
```

File: content/overview/get-started/gs-streams.md [hunk 5]
```diff
@@ -12,52 +12,53 @@ After creating your types, you can add streams and associate a type with the str
...
+1. From the **Streams** drop-down list, select **Streams** if it is not already selected. 
+    ![Streams list](images/streams-list.png)
+
+1. Click the ![More options](images/more-options.png) icon on the menu bar and click **Manage Default Permissions**.  
+    The `Manage Default Permissions for New Streams` window opens. The default permissions for streams created in the selected namespace are configured here.
    
-    **Note:** By default, you must be assigned the Tenant Administrator role to change the default permissions. Any changes only affect new streams. It does not change the permissions on already existing steams. 
+    **Note:** By default, you must be assigned the Tenant Administrator role to change the default permissions. Any changes only affect new streams. It does not change the permissions on already existing streams. 
     
     When you are done reviewing the permissions, click **Cancel** to continue.
```
Comment (lasato-OSI) [inline content/overview/get-started/gs-streams.md:28]:
```suggestion
    When you are done reviewing permissions, click **Cancel** to continue.
```

**Pull request merged**